indico/modules/groups/controllers.py

```diff
@@ -140,3 +143,12 @@ class RHGroupDeleteMember(RHLocalGroupBase):
     def _process(self):
         self.group.members.discard(User.get(request.view_args['user_id']))
         return jsonify(success=True)
+
+
+class RHGroupSearch(RHProtected):
+    @use_args({
+        'name': fields.Str()
+    })
+    def _process(self, args):
+        groups = GroupProxy.search(args['name'])
+        return jsonify([{'is_group': group.is_group, 'name': group.name, 'id': group.name} for group in groups])
```

Comment:
why do we need `is_group`? that's always true for a group so let's just set it to `True`